spec/rails_helper.rb

```diff
@@ -6,6 +6,8 @@
 require 'spec_helper'
 require 'rspec/rails'
 require 'capybara/rails'
+require 'paperclip/matchers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.